docs/azure/sdk/snippets/protocol-convenience-methods/AzureCoreConvenience/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Azure.AI.ContentSafety;
+using Azure.Identity;
+
+// Create the client
+var safetyClient = new ContentSafetyClient(
+    new Uri("https://contentsafetyai.cognitiveservices.azure.com/"),
+    new DefaultAzureCredential());
+
+// Call the convenience method
+AnalyzeTextResult result = safetyClient.AnalyzeText("What is Microsoft Azure?");
+
+// Display the results
+foreach (var item in result.CategoriesAnalysis)
```

Comment:
Use the explicit type instead of `var` to make the code easier to follow on Learn